src/routes/rules/RulesView.vue

```diff
@@ -15,12 +15,10 @@
       <v-row class="flex-column align-start mt-5">
         <div>
           <h1 class="gradient-text">
-            Rules of Cuttle
+            {{ `${t('rules.rulesTitle')}` }}
```

Comment:
We don't need the template string literals inside the `{{ }}` because it's already interpreted as a variable. Those can be useful for concatenating strings but when we just have one translation we're applying we can remove the `'s and ${}.
```suggestion
            {{ t('rules.rulesTitle') }}
```

This applies to every translation we have inside `<template>` tags

Reply:
Resolved in commit 0584c1c